web/main.go

```diff
@@ -1,37 +1,49 @@
 package main
 
 import (
+	"github.com/labbsr0x/goh/gohtypes"
+	"github.com/labbsr0x/whisper-client/client"
 	"github.com/sirupsen/logrus"
 	"net/http"
-
-	"github.com/gorilla/mux"
-	"github.com/labbsr0x/goh/gohtypes"
+	"strings"
 )
 
 func main() {
-	whisperClient := getWhisperClient()
-	whisperToken := getWhisperToken(whisperClient)
-
-	ctx := context{
-		whisperClient: whisperClient,
-		whisperToken:  whisperToken,
+	// retrieve scopes
+	scopesArray := strings.Split(scopes, ",")
+	for _, scope := range scopesArray {
+		scope = strings.Trim(scope, " ")
```

Comment:
```suggestion
		scope = strings. TrimSpace(scope)
```

Reply:
@werberson I cannot commit this suggestion because there is a space between . and TrimSpace. But I will apply the suggestion.